DAL/user.DAL.js

```diff
@@ -0,0 +1,22 @@
+const User = require("./../modules/user.module");
+
+const addNewUser = async (user) => {
+  const newUser = User.create(user);
```

Comment:
`User.create(user)` is a static method? else, upperCased var name should be used with class names only.

Reply:
from what i saw online, mongoose model is stated with uppercase. Is it not true?

Reply:
My bad, missed the context